recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Correct. You want to do everything via pull requests because anything that gets pushed to a branch on the feedstock repo pushes new binaries to anaconda.org and the live conda-forge channel.